src/framework/core/js/FluidDocument.js

```diff
@@ -151,22 +151,27 @@ var fluid_3_0_0 = fluid_3_0_0 || {}; // eslint-disable-line no-redeclare
         return event.target;
     };
 
-    // These function (fluid.focus() and fluid.blur()) serve several functions. They should be used by
-    // all implementation both in test cases and component implementation which require to trigger a focus
-    // event. Firstly, they restore the old behaviour in jQuery versions prior to 1.10 in which a focus
-    // trigger synchronously relays to a focus handler. In newer jQueries this defers to the real browser
-    // relay with numerous platform and timing-dependent effects.
-    // Secondly, they are necessary since simulation of focus events by jQuery under IE
-    // is not sufficiently good to intercept the "focusin" binding. Any code which triggers
-    // focus or blur synthetically throughout the framework and client code must use this function,
-    // especially if correct cross-platform interaction is required with the "deadMansBlur" function.
```

Comment:
Probably worth restoring the last line of this comment which I imagine is still valid - "Any code which ..."